src/plugins/telemetryTable/components/TableComponent.vue

```diff
@@ -498,11 +498,17 @@ export default {
   },
   created() {
     this.filterTelemetry = _.debounce(this.filterTelemetry, 500);
+    const throttledFunc = _.throttle(
```

Comment:
Call throttle callback on leading-edge so that screen update happens in same event loop iteration as telemetry receipt. Improves debugging and also means telemetry is not delayed an additional second beyond the delay created by telemetry batching.